code/modules/materials/Mat_Materials.dm

```diff
@@ -574,6 +574,34 @@ ABSTRACT_TYPE(/datum/material/metal)
 		setProperty("hard", 1)
 
 
+/datum/material/metal/veranium
+	mat_id = "veranium"
+	name = "veranium"
+	desc = "It looks to be sparking."
+	color = "#8effdd"
+
+	New()
+		..()
+		setProperty("electrical", 4)
+		setProperty("thermal", 6)
+		setProperty("density", 4)
+		setProperty("chemical", 2)
+		addTrigger(TRIGGERS_ON_LIFE, new /datum/materialProc/veranium_life())
+
+/datum/material/metal/voltite
+	mat_id = "voltite"
+	name = "voltite"
+	desc = "Energy seems to be flowing around it, chanelled through in an unknown manner."
+	color = "#389fff"
+
+	New()
+		..()
+		setProperty("electrical", 10)
```

Comment:
*Nothing* else has 10 for property values aside from plasma offgas. I get it is trying to be an improvement, but if this is meant to provide a benefit to things perhaps make them react to shock/arcflash instead, with the unique material properties of shocking/arcflashes providing that regularly.